src/components/sign-in-screen/sign-in-screen.jsx

```diff
@@ -1,89 +1,70 @@
-import React, {PureComponent, createRef} from "react";
+import React from "react";
 import PropTypes from "prop-types";
-import {connect} from "react-redux";
+
 import Logo from "../logo/logo";
 import Footer from "../footer/footer";
-import {login} from "../../store/api-actions/api-actions";
-
-class SignInScreen extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.loginRef = createRef();
-    this.passwordRef = createRef();
-
-    this.handleSubmit = this._handleSubmit.bind(this);
-  }
-
-  _handleSubmit(evt) {
-    const {onSubmit} = this.props;
 
-    evt.preventDefault();
+const SignInScreen = (props) => {
+  const {isValid, loginRef, onSubmit, passwordRef} = props;
 
-    onSubmit({
-      login: this.loginRef.current.value,
-      password: this.passwordRef.current.value,
-    });
-  }
+  return (
+    <div className="user-page">
+      <header className="page-header user-page__head">
+        <Logo linkClassName={`logo__link`}/>
+        <h1 className="page-title user-page__title">Sign in</h1>
+      </header>
 
-  render() {
-    return (
-      <div className="user-page">
-        <header className="page-header user-page__head">
-          <Logo linkClassName={`logo__link`}/>
-          <h1 className="page-title user-page__title">Sign in</h1>
-        </header>
-
-        <div className="sign-in user-page__content">
-          <form
-            action=""
-            className="sign-in__form"
-            onSubmit={this.handleSubmit}
-          >
-            <div className="sign-in__fields">
-              <div className="sign-in__field">
-                <input
-                  className="sign-in__input"
-                  type="email"
-                  placeholder="Email address"
-                  name="user-email"
-                  id="user-email"
-                  ref={this.loginRef}
-                />
-                <label className="sign-in__label visually-hidden" htmlFor="user-email">Email address</label>
-              </div>
-              <div className="sign-in__field">
-                <input
-                  className="sign-in__input"
-                  type="password"
-                  placeholder="Password"
-                  name="user-password"
-                  id="user-password"
-                  ref={this.passwordRef}
-                />
-                <label className="sign-in__label visually-hidden" htmlFor="user-password">Password</label>
+      <div className="sign-in user-page__content">
+        <form
+          action=""
+          className="sign-in__form"
+          onSubmit={onSubmit}
+        >
+          {
+            isValid
+              ? ``
+              : <div className="sign-in__message">
+                <p>Please enter a valid email address</p>
```

Comment:
Здечь тоже ренартик лишний и многострочный JSX желательно выделять в круглые скобки
```suggestion
            !isValid && (
              <div className="sign-in__message">
              <p>Please enter a valid email address</p>
```